src/core.js

```diff
@@ -24,7 +38,8 @@ class Core extends Model {
 
 		} else {
 
-			this.data = null; // new InfoboxData(...);
+			// If no 'from' is not defend, we instantiate a fresh infobox
```

Comment:
actually two typos. I think you mean:  `If 'from' is not defined, we instantiate a fresh infobox`